lib/charms/operator_libs_linux/v0/apt.py

```diff
@@ -1289,11 +1309,12 @@ def __init__(self):
                 if not os.path.isfile(default_sources):
                     raise
 
-        # read sources.list.d
-        for file in glob.iglob(os.path.join(sources_dir, "*.list")):
-            self.load(file)
-        for file in glob.iglob(os.path.join(sources_dir, "*.sources")):
-            self.load_deb822(file)
+        with tracer.start_as_current_span("load sources"):
```

Comment:
Same question as above - would you mind explaining why this gets a span? I'm not saying it shouldn't, just trying to better understand.